ui-tests/src/test/java/org/openmrs/reference/CheckModulesTest.java

```diff
@@ -15,7 +15,7 @@
     /**
      * Check the list of modules to be sure they are all started.
      */
-    @Ignore
+    @Test
     @Category(org.openmrs.reference.groups.BuildTests.class)
```

Comment:
You can make it as @Category(buildTest.class) not org.openmrs.reference.groups.buildTests.class simply because the full package is being defined in pom class

Reply:
> @category(buildTest.class)

I think @sherrif10 meant `@Category(BuildTest.class) `

Reply:
I've fixed it in the second commit @kdaud @sherrif10 . kindly review

Reply:
Thanks , Remove **org.openmrs.reference.groups.BuildTests.class** and replace it with **BuildTests.class**